packages/utils/adapters/src/adapters.android.ts

```diff
@@ -2,7 +2,7 @@ import { TextProps, TextPropsIOS, ViewProps, ViewPropsIOS, ImageProps, ImageProp
 import { IFilterMask } from './filter.types';
 
 // export core interface types
-export type ITextProps = Omit<TextProps, keyof TextPropsIOS>;
+export type ITextProps = Omit<TextProps, keyof TextPropsIOS | 'onPressIn' | 'onPressOut'>;
```

Comment:
Why omit these?

Reply:
I think they should be included. These are Pressable style props and I think there was work in RNW to support these for their Text component. Does it cause build errors of we include these? If so, we could maybe mark it as a follow-up

Reply:
Ahh I see. Yeah they did cause type build errors. I'll mark it as a follow up. Thanks for pointing that out!